### ISSUE TITLE
Fixinator client returns inaccurate error messaging when used with enterprise server

### ISSUE THREAD

**Issue by brendanwebb** [foundeo/fixinator#3]
When using the Fixinator client with an enterprise server, the error messaging is inaccurate.

In the following example, I have...

- nginx configured to return a 403 unless a valid x-api-key header is provided
- the client configured to use myserver.com
- provided an invalid API key

And I get the following error message: 

```
Fixinator API Server: https://myserver.com/scan/

 ✓ | Scanning /mycode/

---- Fixinator Client Error ----
Fixinator API Key (xxx) is invalid, disabled or over the API request limit. Please contact Foundeo Inc. for assistance. Please provide your API key in correspondance. https://foundeo.com/contact/ 
403 Forbidden <html>
<head><title>403 Forbidden</title></head>
<body>
<center><h1>403 Forbidden</h1></center>
<hr><center>nginx/1.17.2</center>
</body>
</html>
```

Perhaps if the client is set to use an API server other than the Foundeo hosted one, it could return some more generic messaging, allowing the customization to appear in a customized 403 response.

**Comment by pfreitag** [comment]
Hi @brendanwebb, I'm unclear about what should be corrected. It is saying that _Fixinator API Key (xxx) is invalid_ isn't that why the 403 is triggered in your case?

**Comment by brendanwebb** [comment]
Hi @pfreitag,

I was referring specifically to the "Please contact Foundeo Inc. for assistance. Please provide your API key in correspondance. https://foundeo.com/contact/" portion of the message.

With the enterprise server version of Fixinator, we want our users to contact us, rather than you, for assistance with API keys.

Thanks,
Brendan

**Comment by pfreitag** [comment]
Ok great, that makes sense, thanks for clarifying!  I will address this in a future update.

**Comment by pfreitag** [comment]
Fixed in a53bc0cc89ff157c2f5b8612db8dd06e43f8ccc0
